lineman/app/components/flash/flash_service.coffee

```diff
@@ -1,14 +1,22 @@
 angular.module('loomioApp').factory 'FlashService', ($rootScope) ->
   new class FlashService
 
-    createFlashLevel = (level) =>
+    SHORT_TIME = 2500
+    LONG_TIME = 2147483645
```

Comment:
Tried using Number.MAX_VALUE / MAX_SAFE_VALUE, but it's too big a value for $timeout. :(